be/src/common/config.h

```diff
@@ -203,7 +203,8 @@ namespace config {
     CONF_Int32(disk_stat_monitor_interval, "5");
     CONF_Int32(unused_rowset_monitor_interval, "30");
     CONF_String(storage_root_path, "${DORIS_HOME}/storage");
-    CONF_Int32(min_percentage_of_error_disk, "50");
+    // BE process will exit if the percentage of error disk reach this value.
+    CONF_Int32(max_percentage_of_error_disk, "50");
```

Comment:
How about set the default to 0 to make it strict, so that we won't miss the broken disks

Reply:
Ok